### ISSUE TITLE
Doesn't Launch

### ISSUE THREAD

**Issue by LaughingMan008** [LTCHIPS/rottexpr#53]
Fresh install of the GOG version. I put the .exe in the folder and click it and nothing happens. No error message, no crash, just nothing. Windows 10

**Comment by bebopcello** [comment]
I had this problem too. You need to have just the DARKWAR files or HUNTBGN files in the same folder as your rott.exe and SDL2 dll files. But it won't launch if you have BOTH the DARKWAR and HUNTBGN files in the same directory, so you can't just put it in the GOG.com install folder itself (which is what I tried too).

**Comment by LTCHIPS** [comment]
I did some quick debug runs, and it appears that rottexpr runs into an undefined reference error when attempting to read the SETUP.ROT file, which in a fresh GOG/Steam install of ROTT is present.

rottexpr doesn't need SETUP.ROT to run, it just needs the files pointed out in the README.

Having said that, I feel quite silly for not trying out rottexpr with a fresh install from Steam/GOG up until this point.

TLDR: As a temp fix, remove SETUP.ROT from your ROTT directory. Then rottexpr should start up.

**Comment by Calinou** [comment]
Duplicate of https://github.com/LTCHIPS/rottexpr/issues/46.

**Comment by TheShich** [comment]
It helps to delete SETUP.ROT with Apogee Throwback Pack and Windows 7. Thanks.
